project/app/Helpers/ToHTMLstorage.php

```diff
@@ -25,8 +25,18 @@ public function itemCollectionToHTMLst(Collection $items): string
             </tr>";
     }
 
-    public function itemToHTML(Item $item): string
+    public function itemToHTML(Item $item, bool $add_form): string
     {
+        $form = "<form method='POST' action='".url('update_item', $item->id)."'>
+                        <input type='hidden' name='_token' value='".csrf_token()."' />
+                        <input type='number' id='add' name='add' style='width: 100px; display: inline-block;'/>
+                        <div style='margin-left: 10px; display: inline-block; border: #0c5460 solid 1px; border-radius: 20px; padding: 10px; width: fit-content; background-color: #edf3fc; box-shadow: #6b7280 1px 1px 1px;'>
+                            <input type='submit' style='font-weight: bold' value='+'/>
```

Comment:
Lekka kosmetyczna rzecz (jak nie zostanie zaimplementowana to i tak zaakceptuje ale): wydaje mi się, że myszka powinna zmieniać się w łapkę/palca wskazującego po najechaniu na plusa

Reply:
Może zrobię